test/src/main/java/org/apache/accumulo/test/upgrade/ConfigPropertyUpgraderIT.java

```diff
@@ -62,13 +65,35 @@ public class ConfigPropertyUpgraderIT {
   private static ZooKeeper zooKeeper;
   private static ZooReaderWriter zrw;
 
+  // Create legacy renamer for this test
+  private static final DeprecatedPropertyUtil.PropertyRenamer MASTER_MANAGER_RENAMER =
+      DeprecatedPropertyUtil.PropertyRenamer.renamePrefix("master.",
+          Property.MANAGER_PREFIX.getKey());
+  private static List<DeprecatedPropertyUtil.PropertyRenamer> renamers;
+
   private InstanceId instanceId = null;
 
   @TempDir
   private static File tempDir;
 
+  static {
+    try {
+      // Make the renamers list visible to be modified
+      final Field renamersField = DeprecatedPropertyUtil.class.getDeclaredField("renamers");
+      renamersField.setAccessible(true);
+      @SuppressWarnings("unchecked")
+      List<DeprecatedPropertyUtil.PropertyRenamer> renamers = (List<
+          DeprecatedPropertyUtil.PropertyRenamer>) renamersField.get(DeprecatedPropertyUtil.class);
+      ConfigPropertyUpgraderIT.renamers = renamers;
+    } catch (Exception e) {
+      throw new RuntimeException(e);
+    }
+  }
```

Comment:
I think we could probably avoid the static initializer stuff and all the type-safety check bypassing if the code we're testing is tweaked slightly to make it easier to test with a method that makes the list visible for testing purposes.

Reply:
I could add a public getter but I also kind of dislike how it's set up in general as the ranamers field should really just be an Immutable list as well. The issue of course is if I make it Immutable then I can't change it for testing. What I might do is make a second static method that takes a list of renamers and the original can delegate so we can pass in whatever we want for testing.

Reply:
Nevermind, I'm just going to go with your original idea to add a method to get the list so it's visible as adding another util method doesn't help since the upgrader logic won't ever call it.